Assignment3Loops/src/assignment3loops/Assignment3Loops.java

```diff
@@ -0,0 +1,33 @@
+package assignment3loops;
+import java.util.Scanner;
+
+public class Assignment3Loops {
+    public static void main(String[] args) {
+        //Keyboard input
+        Scanner keyboard = new Scanner(System.in);
+        int cont;//do-while variable
+        
+        print("Welcome to Steve's word Mixer-upper!\n\n");
+       do{
+        print("Please Enter A word you would "
+                + "Like to mix up: ");        
+        String word = keyboard.nextLine();//User Input
+                print("\nYour word reversed is: ");
+                for(int i = word.length()-1; i >= 0; i--){
+                    System.out.print(word.charAt(i));  
+                }
+                
+        print("\n\nIf you would like to continue press 1, if not press 0: ");
+        cont = keyboard.nextInt();
+        //to clear the buffer.
+        keyboard.nextLine();
+        print("\n");
+       
+       }
+       while(cont == 1);
+    }
+    //print method
+    public static void print(String str){
+     System.out.print(str);
+ } 
+}
```

Comment:
I like the wrapper with the do-while to keep it going as long as the user wants to.